### ISSUE TITLE
Help page in project builder that explains format and content of the export files

### ISSUE THREAD

**Issue by mschwamb** [zooniverse/Panoptes-Front-End#3474]
There is currently no help content linked from the project builder for the data exports. The exports recently changed format slightly, and all my codes broke that flatten the json to a csv with the columns I need for my analysis. I had go through the data exports and figure out what columns had been removed  and what new columns that had been added and what value types they contain. At least one column I'm currently unclear about what information it provides as  the column appears to be empty but it's new in the export. 

I fully understand that the data format for the project builder output could change without warning as it is still in development , but it would really help project owners if there was a page in the project builder that had the information of what each column of the export files means for  project owners.  I think this would also be helpful for new project builders who are encountering the project builder and its output for the first time, as well as project owners developing their own processing routines.

**Comment by mschwamb** [comment]
I still think this is important. I'm still parsing the exports for my collaborations because the other science team members have too much of a learning curve on understanding the data export format and values exported.

**Comment by mschwamb** [comment]
@camallen not really. It's a bit like jumping into the deep end. Great if you have some sense of the data, but even these codes aren't explaining  fully what is actually in the data dumps even the comments or complete. Also it's reading someone else's code without example data to actually play with it and figure out what you don't understand from just reading code. 

so I can kinda of read this https://github.com/zooniverse/Data-digging/blob/master/example_scripts/exoplanet_explorers/aggregate_planetclass.py 

but there's alot I'm not 100% sure about and I can't actually run it on some Exoplanet Explorers output to figure out what I don't understand

Also  this file: https://github.com/zooniverse/Data-digging/tree/master/docs - hasn't been edited in two years

> y (float) - the vertical value of the point as a real number relative to *** what? *** using the 
> ***upper/lower?***  left corner as the origin

> x (float) - the horizontal value of the point as a real number relative to *** what? *** using the 
>  ***upper/lower?***  left corner as the origin

Since clearly no one is updating it. It's not super helpful since there are question marks on the exact kind of info I need to be able to process panoptes P4 tests and compare to original P4 classifications. 

The developers know whether (0,0) is top left corner or bottom left corner but that's not easy for me to figure out as a project builder.  So I'll have to draw a few test marks and play around with plotting to figure out the answer.

**Comment by mschwamb** [comment]
Actually I  disagree I think this should be a page on in the project builder online documents not in a repo. Just like there is a help documents online to make a zooniverse project. I think this part of that and it shouldn't be locked in a github repo it should be easily accessible in the same way the best practices documents are

**Comment by eatyourgreens** [comment]
[Data-digging](https://github.com/zooniverse/Data-digging) already has [a description of the export fields](https://github.com/zooniverse/Data-digging/blob/master/docs/classification_export_field_descriptors.txt) so I'm going to leave this closed.

I'm happy to re-open it and assign it to you if you'd like to take it on?

There's already an open issue about the lack of documentation for the drawing task annotations (#4478.)

**Comment by eatyourgreens** [comment]
>The developers know whether (0,0) is top left corner or bottom left corner but that's not easy for me to figure out as a project builder. So I'll have to draw a few test marks and play around with plotting to figure out the answer.

Actually, we don't know this since the drawing tools aren't documented (#4478). We can take a best guess for you but I usually end up testing it by trying some annotations with [the dev classifier](https://master.pfe-preview.zooniverse.org/dev/classifier) on our staging server.
